internal/store/postgres/migrations/20220819074139_add_metadata_keys_table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS metadata_keys (
+    id           uuid        PRIMARY KEY     DEFAULT uuid_generate_v4(),
```

Comment:
why do we need `id` if we can just use `key`?

Reply:
Yes, the key as the primary key would have worked fine as well. We can change it to key.

One thing I am not sure about is, let's say we later introduce get, update, and delete APIs as well. Would it be fine to use the "key" itself as the identifier or would it be good(safe) to use a UUID there?

Reply:
Removed `id`